core/vm/evm.go

```diff
@@ -103,7 +103,11 @@ func NewEVM(blockCtx evmtypes.BlockContext, txCtx evmtypes.TxContext, state evmt
 	}
 
 	// [zkevm] change
-	evm.interpreter = NewZKEVMInterpreter(evm, vmConfig)
+	zkConfig := ZkConfig{
```

Comment:
to keep it more rebase-friendly, let's have a method like `NewZkConfig` and use it directly in the `evm.interpreter` assignment like that `evm.interpreter = NewZKEVMInterpreter(evm, NewZKConfig(vmConfig))` here and in another place.